plugins/Wallet/js/components/newwalletdialog.js

```diff
@@ -7,13 +7,9 @@ const NewWalletDialog = ({password, seed, showConfirmationDialog, actions}) => {
 			{showConfirmationDialog ? <ConfirmationDialog /> : null}
 			<p> You have created a new wallet!  Please write down the seed and password in a safe place.  If you forget your password, you won't be able to access your wallet. </p>
 			<h2> Seed: </h2>
-			<span className="newwallet-seed">
-				{seed}
-			</span>
+				<div id="new-wallet-seed"> <strong> {seed} </strong> </div>
```

Comment:
Linting is complaining about this line and the one before, just need to remove the one tab of spacing and the test pass (verified on my machine). Looked at the new styling and it looks good.